lib/receipt.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'product'
+
+class Receipt
+  attr_reader :business, :address, :phone, :items, :tax, :total
+
+  def initialize
+    @business = ''
+    @address = ''
+    @phone = ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.